src/components/site/FAQpage/FAQPage.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import Styled from '@emotion/styled';
+import { NavLink} from 'react-router-dom';
+
+
+
+const H1=Styled.h1`
+text-align:center;
+text-decoration: underline;
+margin-bottom:10px;
+font-family:'Ubuntu', sans-serif;
+`;
+const H2=Styled.h2`
+text-align:center;
+margin-bottom:15px;
+font-family:'Ubuntu', sans-serif;
+
+`;
+const Footer = Styled.footer`
+display: flex;
+    justify-content: center;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    background-color: rgb(46, 125, 245);
+    color: white;
+    text-align: center;
+    padding: 8px;
+    font-family:'Ubuntu', sans-serif;
+`;
+const A=Styled.a`
+text-decoration: none;
+color: white;
+font-family:'Ubuntu', sans-serif;
+`;
+const Div=Styled.div`
+display:flex;
+justify-content: center;
+align-items: center;
+flex-direction: column;
+flex-wrap: wrap;
+justify-content: space-around;
+`;
+const Button=Styled.button`
+ width: 500px;
+ height: 60px;
+ margin-bottom:15px;
+ font-family:'Ubuntu', sans-serif;
+`;
+const Navbar=Styled.div`
+box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.28);
+padding: 15px 10px 10px 10px;
+background-color: rgb(12, 104, 243);
+display: flex;
+justify-content: center;
+align-items: center;
+margin-bottom: 0%;
+`;
+const Ul =Styled.ul`
+    display: flex;
+    margin-left: 700px;
+    margin-right: 100px;
+`;
+const Li =Styled.li`
+    padding-left: 8px;
+    list-style: none;
+    margin-left: 10px;
+`;
+const Span=Styled.span`
+    color: #fff;
+    font-size: 17px;
+    text-decoration: none;
+    font-family:'Ubuntu', sans-serif;
+`;
+
+const Faqs = () =>{
+ 
+  return(
+    <div>
+      <Navbar>
+        <h1><NavLink className="logo" to="/">Phantom Ride</NavLink></h1>
+        <Ul>
+          <Li>
+            <NavLink to="/Contact"><Span>Contact Us</Span></NavLink>
+          </Li>
+          <Li>
+            <NavLink to="/aboutPage"><Span>About Us</Span></NavLink>
+          </Li>
+          <Li>
+            <NavLink to="/faqs"><Span>FAQs</Span></NavLink>
+          </Li>
+        </Ul>
+      </Navbar>
+      <H1><span>FAQs</span></H1>
+      <H2>Frequently Asked questions</H2>
+      <Div>
+        <Button >How can I get information about Phantom app </Button>
+        
+        <Button>How can I get information about Phantom app</Button>
+        
+        <Button>How can I get information about Phantom app</Button>
+        
+        <Button>How can I get information about Phantom app</Button>
+        
+      </Div>
+      <Footer>
+        <p>&copy; Phantom Ltd</p>
+        <A href="#" >Facebook</A>
+        <A href="#" >Twitter</A>
+        <A href="#" >Gmail</A>
+        <A href="#" >Instagram</A>
+      </Footer>
```

Comment:
The  `<Footer>` should be refactored to its own component should that it can be shared across multiple pages.

Reply:
I have changed it